gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/model/mapper/FlowMigration.java

```diff
@@ -26,13 +29,16 @@
 import java.util.Collection;
 import java.util.List;
 import java.util.Set;
+import java.util.function.UnaryOperator;
 import java.util.stream.Stream;
 import lombok.extern.slf4j.Slf4j;
 import org.jspecify.annotations.Nullable;
 
 @Slf4j
 public class FlowMigration {
 
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
We are creating ObjectMapper in "a lot of classes" Could we rely on the singleton bean created [here](https://github.com/gravitee-io/gravitee-api-management/blob/612b7056513ccb82af4b00ad418ffdd053ccaef5/gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/infra/json/jackson/JacksonSpringConfiguration.java#L41) ?

Also using jackson classes directly in the core domain is not great but not doing that would be more complicated and I'm not sure about the benefits